pybossa/data_access.py

```diff
@@ -247,7 +247,10 @@ def valid_user_type_based_data_access(user_type, access_levels):
 @when_data_access()
 def set_default_amp_store(project):
     annotation_config = project['info'].get('annotation_config', {})
-    input_data = project['info'].get('input_data') or ''
-    output_data = project['info'].get('output_data') or ''
-    annotation_config['amp_store'] = ('L1' in str(input_data) + str(output_data) == False)
+    input_data = project['info'].get('data_classification', {}).get('input_data') or ''
+    output_data = project['info'].get('data_classification', {}).get('output_data') or ''
+    if 'L1' in (str(input_data) + str(output_data)):
```

Comment:
Could we add a comment at this line (since this is business-specific logic):
```python
# Set amp_store default checkbox to False if 'L1' is in input or output data classifications.
```

Reply:
sure